tip-wlan/charts/wlan-cloud-graphql-gw/templates/ingress.yaml

```diff
@@ -32,6 +32,12 @@ spec:
     - host: {{ .host | quote }}
       http:
         paths:
+        {{- if $.Values.ingress.alb_https_redirect }}
```

Comment:
As @leonidlm pointed earlier, calling this flag alb_https_redirect would make it AWS specific. If this feature of alb is also present in other cloud platforms, perhaps renaming it to `lb_https_redirect` would be better. What do you think?

Reply:
interesting idea, yeah why not, I am pretty sure azure or GCP won't require this but making it a little generic will not hurt